examples/explorer/README.md

```diff
@@ -0,0 +1,29 @@
+# Basic Explorer
+
+A simple Cardano Explorer using Reactjs, React-bootstrap and Relay to use with Jorgunmandr node.
+
+# Prerequesites
+
+- You need a Jormungandr node running in explorer mode in order to use this app.
+
+# Install dependencies
+
+- Use nodejs v8.12: `nvm install 8.16.0 && nvm use 8.16.0`
+- Node dependencies: `npm install`
+
+# Configure
+
+- Put the Jormungandr's Explorer API URL into the configuration file in `src/config.json` in the `explorerUrl` key.
```

Comment:
link to the jormungandr documentation here too: https://input-output-hk.github.io/jormungandr/quickstart/03_rest_api.html